### ISSUE TITLE
About G/D loss function

### ISSUE THREAD

**Issue by zicair** [batsa003/salgan#10]
Thanks for you code, In Paper, the loss function of 'Generator' formulated as
 ![image](https://user-images.githubusercontent.com/41500153/54907648-ff29fb80-4f20-11e9-9214-25aec28b8f12.png)
And the loss function of 'Discriminator' formulated as
![image](https://user-images.githubusercontent.com/41500153/54907886-94c58b00-4f21-11e9-9f0a-9c4e60019540.png)
they both use BCE loss; I found out that you didn't do it that way,you used log(output) in 'Discriminator' and -log(output) in 'Generator' and I was confused about that! Can you explain it for me.Thanks a lot.

**Comment by lyq893** [comment]
@zicair 
hello!
Glad to find you following this code.
I want to know how does this train the model? How does it work in the first place? Can you answer my question?
Looking forward to your answer! thanks.
